src/services/index.js

```diff
@@ -1,19 +1,11 @@
-'use strict';
 
-const requireDirectory = require('require-directory');
```

Comment:
Should we remove this from dependencies?

Reply:
I dont know what it is, it was on the initial code generated via yeoman generator.
But We can also delete this dep from package.json

Reply:
I added it to load all the services without having to "register" them here each time, but it felt a little hacky. I'll remove it :)